package.json

```diff
@@ -2,14 +2,15 @@
   "name": "python-bridge",
   "version": "1.1.0",
   "description": "Node.js to Python bridge ✨🐍🚀✨",
-  "main": "index.js",
+  "main": "index-compiled.js",
   "types": "index.d.ts",
   "scripts": {
     "lint": "npm run lint:ts",
-    "lint:ts": "tslint --project tsconfig.json --type-check",
+    "lint:ts": "tslint --project tsconfig.json",
     "test": "npm run lint && npm run test:js && npm run test:ts",
-    "test:js": "tap test.js",
-    "test:ts": "tsc --lib ES2015 test_typescript.ts && tap test_typescript.js"
+    "test:js": "tap test-compiled.js",
+    "test:ts": "tsc --lib ES2015 test_typescript.ts && tap test_typescript.js",
+    "compile": "babel index.js --out-file index-compiled.js && babel test.js --out-file test-compiled.js"
```

Comment:
I'd suggest adding a prepare script that runs `npm run compile`.

[Here's a good article](https://blog.jim-nielsen.com/2018/installing-and-building-an-npm-package-from-github/) explaining how it works.

Reply:
This removes the need to store a compiled version in git.